CalorimeterGeom/inc/CaloInfo.hh

```diff
@@ -21,7 +21,6 @@ namespace mu2e {
     {
        public:     
           CaloInfoData() : data_() {};
-          ~CaloInfoData() {};
 
           const T& get(const std::string& key) const
```

Comment:
How often is this text-based lookup called?  Is it ever in a tight loop (digis)?  If so, providing an explicit accessor will be much more efficient.

Reply:
Mostly when constructing objects / classes. There are two methods that are called more often to get the number of SiPMS and the crystal length. None seem to be in a tight look, though KalFit calls them to add hits. I would prefer to keep the interface clean, but I can make an exception if this is worrisome